ivy/__init__.py

```diff
@@ -276,6 +277,25 @@ class Node(str):
 
 invalid_devices = ("gpu", "tpu")
 
+# supported ivy data types
+all_ivy_dtypes_str = (
```

Comment:
Instead of creating a new list/tuple of dtypes, we could just directly use `all_dtypes` present in this file. Given that all our `Dtype` classes extend `str`, `dtype_str not in ivy.all_dtypes` or  `dtype_str not in all_dtypes` in this case would also work.